internals/webpack/webpack.base.babel.js

```diff
@@ -13,7 +13,7 @@ module.exports = (options) => ({
   }, options.output), // Merge with env dependent settings
   module: {
     loaders: [{
-      test: /\.js$/, // Transform all .js files required somewhere with Babel
+      test: /\.js(on)?$/, // Transform all .js, .json files required somewhere with Babel
```

Comment:
again, not to be picky at all but json shouldn't need to be transpiled, we should use json-loader